cli/serve/serve.go

```diff
@@ -192,8 +194,52 @@ func serverMain(args []string, c cli.Config) error {
 	registerHandlers()
 
 	addr := net.JoinHostPort(conf.Address, strconv.Itoa(conf.Port))
-	log.Info("Now listening on ", addr)
-	return http.ListenAndServe(addr, nil)
+
+	endpointCA := conf.EPCAFile
+	endpointKey := conf.EPCAKeyFile
+
+	if endpointCA == "" || endpointKey == "" {
+		log.Info("Now listening on ", addr)
+		return http.ListenAndServe(addr, nil)
+	}
+	var CipherSuites = []uint16{
```

Comment:
I have found default ciphersuites are not all supported by Safari browser, but I will double check the default set.